.circleci/config.yml

```diff
@@ -5,15 +5,17 @@ orbs:
 
 jobs:
   validate:
-    executor: hmpps/java
+    executor:
+      name: hmpps/java
+      tag: "11.0"
```

Comment:
Whilst not strictly necessary I have set this to 11.0 in case the ORB default happens to change.  This also keeps it in line with the base image used below **cimg/openjdk:11.0**.